benchexec/test_core_assignment.py

```diff
@@ -37,13 +38,13 @@ def setUpClass(cls):
         cls.longMessage = True
         logging.disable(logging.CRITICAL)
 
-    def assertValid(self, coreLimit, num_of_threads, expectedResult=None):
-        result = _get_cpu_cores_per_run0(coreLimit, num_of_threads, *self.machine())
+    def assertValid(self, coreLimit, num_of_threads, with_hyperthreading, expectedResult=None):
```

Comment:
Would it make sense to use `self.use_ht` instead of adding the parameter `with_hyperthreading`? That would mean we would not need to add a parameter to all existing tests.

Or is there some difference between the attribute and the parameter, e.g., are they set differently at some point?

Reply:
Yes using `with_hyperthreading` with `self.use_ht` is  redundant actually. I shall fix them asap.